wls-infomanagement-service/src/test/java/de/muenchen/oss/wahllokalsystem/infomanagementservice/rest/wahltag/KonfigurierterWahltagControllerIntegrationTest.java

```diff
@@ -0,0 +1,288 @@
+            // DB leer, Wahltag neu setzen mit WahltagStatus.AKTIV -> OK zurück mit empty body
+            val konfigurierterWahltagDTO = new KonfigurierterWahltagDTO(LocalDate.now(), "1-2-3", WahltagStatus.AKTIV, "4711");
+            val requestPost = createPostWithBody(konfigurierterWahltagDTO);
+
+            val responsePost = api.perform(requestPost).andExpect(status().isOk()).andReturn();
+
+            Assertions.assertThat(responsePost.getResponse().getContentAsString()).isEmpty();
+
+            // vorher gesetzten Wahltag aus DB lesen -> OK zurück mit vorher gesetztem body
+            val requestGet = MockMvcRequestBuilders.get("/businessActions/konfigurierterWahltag");
+            val responseGet = api.perform(requestGet).andExpect(status().isOk()).andReturn();
+            val responseGetBody = objectMapper.readValue(responseGet.getResponse().getContentAsString(), KonfigurierterWahltagDTO.class);
+
+            val expectedResponseGetBody = new KonfigurierterWahltagDTO(konfigurierterWahltagDTO.wahltag(), konfigurierterWahltagDTO.wahltagID(),
+                    konfigurierterWahltagDTO.wahltagStatus(), konfigurierterWahltagDTO.nummer());
+
+            Assertions.assertThat(responseGetBody).isEqualTo(expectedResponseGetBody);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_POST_KONFIGURIERTERWAHLTAG, Authorities.REPOSITORY_WRITE_KONFIGURIERTERWAHLTAG,
+                        Authorities.SERVICE_GET_KONFIGURIERTERWAHLTAG, Authorities.REPOSITORY_READ_KONFIGURIERTERWAHLTAG }
+        )
+        void newInaktivWahltagSavedAndReadOKNoContent() throws Exception {
+            // Dooffall: DB leer neu setzen mit WahltagStatus.INAKTIV -> OK zurück mit empty body
+            val konfigurierterWahltagDTO = new KonfigurierterWahltagDTO(LocalDate.now(), "1-2-3", WahltagStatus.INAKTIV, "4711");
+            val requestPost = createPostWithBody(konfigurierterWahltagDTO);
+
+            val responsePost = api.perform(requestPost).andExpect(status().isOk()).andReturn();
+
+            Assertions.assertThat(responsePost.getResponse().getContentAsString()).isEmpty();
+
+            // vorher gesetzten Wahltag lesen -> OK zurück mit empty body weil kein aktiver Wahltag gefunden
+            val requestGet = MockMvcRequestBuilders.get("/businessActions/konfigurierterWahltag");
+            val responseGet = api.perform(requestGet).andExpect(status().isNoContent()).andReturn();
+
+            Assertions.assertThat(responseGet.getResponse().getContentAsString()).isEmpty();
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_POST_KONFIGURIERTERWAHLTAG, Authorities.REPOSITORY_WRITE_KONFIGURIERTERWAHLTAG,
+                        Authorities.SERVICE_GET_KONFIGURIERTERWAHLTAG, Authorities.REPOSITORY_READ_KONFIGURIERTERWAHLTAG }
+        )
+        void overrideAktivWahltagAndReadOKWithContent() throws Exception {
```

Comment:
Vorschlag für ein besseres Naming

```suggestion
        void overrideAktiverWahltagAndReadOKWithContent() throws Exception {
```

oder die englische Variante